src/components/DropdownButton/DropdownButton.tsx

```diff
@@ -0,0 +1,35 @@
+import clsx from 'clsx';
+import React, { ReactNode, forwardRef } from 'react';
+import styles from './DropdownButton.module.css';
+import Icon from '../Icon';
+
+type Props = {
+  className?: string;
+  text?: ReactNode;
+};
+
+/**
+ * ```ts
+ * import {DropdownButton} from "@chanzuckerberg/eds-components";
+ * ```
+ *
+ * A styled button with an expand icon to be used in triggering Popovers, Dropdowns, etc.
+ */
+export const DropdownButton = forwardRef<HTMLButtonElement, Props>(
+  ({ className, text, ...rest }, ref) => {
+    return (
+      <button
+        className={clsx(styles['dropdown-button'], className)}
+        ref={ref}
+        {...rest}
+      >
+        {/* Wrapping span ensures that `children` and icon will be correctly pushed to
+            either side of the button even if `children` contains more than one element. */}
+        <span>{text}</span>
+        <Icon name="expand-more" purpose="decorative" size="1.25rem" />
```

Comment:
This isn't a regression, but I'm just wondering – should we be rotating thing icon on open/close? It's kinda weird that it's always pointed down or whatever, right? Example in the new `Accordion`: https://github.com/FB-PLP/traject/blob/c344cc8526a5d1d1a957e17bcb0779e56da58508/app/assets/javascripts/v2/core/EDSCandidates/Accordion/Accordion.tsx#L82

Reply:
Mmmm good q, I'll rotate the icon when opening/closing

Reply:
Mmmm good q, I'll rotate the icon when opening/closing

Reply:
Rotated in https://github.com/chanzuckerberg/edu-design-system/pull/966/files#diff-dd4dd3af268081f171000b5868e6a442be7aa697fe47f8f999e6d456020fc1ccR30-R45, lmk if you think the rotation is too fast (available tokens were 150ms or 400ms)

Reply:
>lmk if you think the rotation is too fast (available tokens were 150ms or 400ms)

I definitely prefer 150 to 400, but 150 does feel a bit fast. What if we added a third token between them (like `--eds-anim-move-medium`) and set it to 250 or 300?

Reply:
I'm down to do that, will add a 300 token and use that